CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+
+## v0.33.9
+- Upgrade Ubuntu version to 20.04 to be current. 
+=======
 ## next
```

Comment:
The version `v0.33.9` is already part of the CHANGELOG. This change could be added under the `next` section for now and could be part of the next release. Thoughts?

Reply:
sounds good to me.

Reply:
On a side note, I realized that `v0.33.9` is on the CHANGELOG, but there was no release created